DOCS-2305: Email corrections (#1306)
MultiSafepay/docs: content/tools/flexible_3d/_index.md

For more information about 3D Secure, see [About 3D Secure](/faq/payment-regulations/about-3d-secure).

{{< alert-notice >}}

> __Please note__: MultiSafepay no longer supports Flexible 3D with merchants based in Europe due to PSD2 regulations. If your business handles many transactions with customers outside of Europe and you are interested in a more flexible setup, email your account manager by email <sales@multisafepay.com>

Reviewer: ```suggestion
**Note:** MultiSafepay no longer supports Flexible 3D for merchants in Europe due to PSD2 regulations. If you have a lot of customers outside Europe and you need a more flexible setup, email your account manager at <sales@multisafepay.com>
```
